src/main/webapp/app/pages/curation/nestLevelSummary/NestLevelSummary.tsx

```diff
@@ -27,7 +27,7 @@ export const NestLevelSummary = (props: NestLevelSummaryProps) => {
 
   let lastBadgeHasHiddenNumber = false;
 
-  const badges = (
+  const badges = props.summaryStats ? (
```

Comment:
Was there a scenario where `summaryStats` was undefined? If you look in firebase.gene.store.ts, the `mutationLevelMutationSummaryStats` returns at minimum an empty object

Reply:
Yeah, for some of the genes with more mutations, the mutationLevelSummaryStats were undefined. I believe it's because the calculation was taking a little longer, so that empty object wasn't even returned yet.